x/ecocredit/client/testsuite/query.go

```diff
@@ -160,10 +160,58 @@ func (s *IntegrationTestSuite) TestQueryClassCmd() {
 }
 
 func (s *IntegrationTestSuite) TestQueryBatchesCmd() {
-	val := s.network.Validators[0]
-	clientCtx := val.ClientCtx
-	clientCtx.OutputFormat = "JSON"
-	_, projectName, batchDenom := s.createClassProjectBatch(clientCtx, val.Address.String())
+	ctx := s.val.ClientCtx
+	ctx.OutputFormat = "JSON"
+
+	testCases := []struct {
+		name           string
+		args           []string
+		expectErr      bool
+		expectedErrMsg string
+	}{
+		{
+			name:           "too many args",
+			args:           []string{"foo"},
+			expectErr:      true,
+			expectedErrMsg: "Error: accepts 0 arg(s), received 1",
+		},
+		{
+			name: "valid with pagination",
+			args: []string{
+				fmt.Sprintf("--%s", flags.FlagCountTotal),
+			},
+			expectErr: false,
+		},
+	}
+
+	for _, tc := range testCases {
+		s.Run(tc.name, func() {
+			cmd := coreclient.QueryBatchesCmd()
+			out, err := cli.ExecTestCLICmd(ctx, cmd, tc.args)
+			if tc.expectErr {
+				s.Require().Error(err)
+				s.Require().Contains(out.String(), tc.expectedErrMsg)
```

Comment:
not sure if we bumped the `require` package version yet, but latest should have a `ErrorContains` which does both of these in one call

Reply:
`ErrorContains` requires an `error` type and need to check the json output here so we need to use `Contains`.

Reply:
derp. just realized the args to `Contains` 🤦🏻